tonic-types/src/lib.rs

```diff
@@ -13,8 +23,391 @@
 #![doc(html_root_url = "https://docs.rs/tonic-types/0.6.0")]
 #![doc(issue_tracker_base_url = "https://github.com/hyperium/tonic/issues/")]
 
-mod pb {
+use prost::{DecodeError, Message};
+use prost_types::Any;
+use tonic::{codegen::Bytes, Code};
+
+/// Useful protobuf types
+pub mod pb {
     include!(concat!(env!("OUT_DIR"), "/google.rpc.rs"));
 }
 
 pub use pb::Status;
+
+mod error_details;
+mod error_details_vec;
+mod std_messages;
+
+pub use std_messages::*;
+
+pub use error_details::ErrorDetails;
```

Comment:
Yes, this indeed looks better. But besides that, should we go a bit further in mod reorganization and add all new functionality to a `richer_error` mod? This would probably make easier to add support to other things into `tonic-types` in future PR's. I should have considered this before. I also could adjust the code to use `mod.rs`, to make the style more consistent across crates

Reply:
Yeah, though we could do this in a follow up PR if that works.

Reply:
Sure. In the latest commit I just moved `error_details_vec` into `error_details`.